### PR TITLE
📦  Viceroy 0.2.15 release

### PR DESCRIPTION
**Pull request by mgattozzi**

### CODE REVIEW

**Comment by mgattozzi** [comment]
Got approval offline. Pushed tags to do a release.

**Review by JakeChampion — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 ## Unreleased
```
Comment (acfoltzer) [inline CHANGELOG.md:1]: Is it okay for this header to be here in a release commit?

Comment (acw) [reply CHANGELOG.md:1]: That's what [the release guide told me to do](https://github.com/fastly/Viceroy/blob/main/doc/RELEASING.md). :shrug:

Comment (acfoltzer) [reply CHANGELOG.md:1]: Ohh the release has already happened, this is approval for the merge. Gotcha!

**Review by acfoltzer — APPROVED** [review]

**Pull request merged**